Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan_-_Quarantine_File.yml

```diff
@@ -0,0 +1,427 @@
+id: Containment Plan - Quarantine File
+version: -1
+name: Containment Plan - Quarantine File
+description: |-
+  This playbook is one of the sub-playbooks in the containment plan.
```

Comment:
Maybe give references and break it into sections.

"This playbook....is part of the [Containment Plan](link) playbook.

Same here, the second sentence isn't clear.